src/representation_theory/basic.lean

```diff
@@ -19,17 +20,17 @@ including equivalences between different definitions of representations.
   * `representation.as_module`
   * `representation.as_group_hom`
   * `representation.character`
+  * `representation.tensor_rep`
 
 ## Implementation notes
 
 A representation of a monoid `G` over a commutative semiring `k` is implemented as a `k`-module `V`
 together with a `distrib_mul_action G V` instance and a `smul_comm_class G k V` instance.
-
-Alternatively, one can use a monoid homomorphism `G →* (V →ₗ[k] V)`. The definitions `as_monoid_hom`
-and `rep_space` allow to go back and forth between these two definitions.
 -/
 
 open monoid_algebra
+open linear_map
+open distrib_mul_action
```

Comment:
It might be better to just use
```suggestion
open distrib_mul_action (to_module_End)
```
Rather than pulling in random other lemmas that I assume you're not using?

Reply:
I also use `to_linear_map`, `to_linear_map_apply`,  and `to_module_End_apply`, so I thought it worth opening. Are there potential downsides to opening a namespace?

Reply:
Occasionally things get confusing with too many namespaces open, especially if not enough things are `protected`. I hadn't noticed you were using all three here though. So that's fine.